### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: docs/reference/orm.rst [hunk 1]
```diff
@@ -149,7 +149,7 @@ Example for configure search filter
 
     final class FAQCategoryAdmin extends AbstractAdmin
     {
-        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+        protected function configureDatagridFilters(DatagridMapper $datagridMapper): void
```
Comment (jordisala1991) [inline docs/reference/orm.rst]:
```suggestion
        protected function configureDatagridFilters(DatagridMapper $filter): void
```

It was filter or filters? I do not remember now.

Comment (VincentLanglet) [reply docs/reference/orm.rst]: $filter

File: docs/reference/orm.rst [hunk 2]
```diff
@@ -149,7 +149,7 @@ Example for configure search filter
 
     final class FAQCategoryAdmin extends AbstractAdmin
     {
-        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+        protected function configureDatagridFilters(DatagridMapper $filter): void
         {
             $datagridMapper
```
Comment (jordisala1991) [inline docs/reference/orm.rst:154]: remember to change this one too.

Comment (VincentLanglet) [inline docs/reference/orm.rst]:
```suggestion
            $filter
```

Comment (VincentLanglet) [reply docs/reference/orm.rst:154]: thanks

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**